distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Zone.enso

```diff
@@ -120,7 +121,10 @@ type Zone
 
              example_zone_id = Zone.system.zone_id
     zone_id : Text
-    zone_id self = self.internal_zone_id . getId
+    zone_id self = @Builtin_Method "Zone.zone_id"
+
+    equals : Zone -> Boolean
```

Comment:
Why there is `equal` as well as `==`? Something related to compatibility with former `asGuestObject(...).equals(...)`? Shall not such compatibility layer be ultimately removed?

Reply:
You are right, `equals` is not necessary anymore